src/App.tsx

```diff
@@ -1,24 +1,215 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect, useState, FormEvent, useMemo,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import {
+  createTodo, deleteTodo, getTodos, updateTodo,
+} from './api/todos';
+import { TodoStatus } from './types/TodoStatus';
+import { ErrorType } from './types/ErrorType';
 
-const USER_ID = 0;
+export const USER_ID = 9929;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<TodoStatus>(TodoStatus.All);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.NoError);
```

Comment:
My assumption is there are no error it should be null insead of enum value. So it will be easier to chech without creating extra variables `isError` - just  `if (errorType) `

Reply:
Agree. Fix it 
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/97547824/3d03a623-e5d3-4e59-82b1-0bb40b963b34)